taskcluster/ac_taskgraph/build_config.py

```diff
@@ -29,7 +31,9 @@ def get_components():
 
 
 def get_version():
-    return ensure_text(_read_build_config()["componentsVersion"])
+    with open(os.path.join(PROJECT_DIR, "version.txt")) as fh:
```

Comment:
Nit: Let's add a comment in `.buildconfig.yml` to say that we have to: 

 a. Manually keep the `componentsVersion` in sync with `version.txt`
 b. Stop using this entry as a reference for version numbers.

This way, we can migrate the rest of the toolchain to use `version.txt` in a followup.

Reply:
done